conans/client/conanfile/configure.py

```diff
@@ -1,44 +1,40 @@
 from conans.errors import conanfile_exception_formatter
 from conans.model.pkg_type import PackageType
 from conans.model.requires import BuildRequirements, TestRequirements, ToolRequirements
-from conans.util.env import no_op
 
 
 def run_configure_method(conanfile, down_options, profile_options, ref):
     """ Run all the config-related functions for the given conanfile object """
 
-    # Avoid extra time manipulating the sys.path for python
-    with no_op():  # TODO: Remove this in a later refactor
```

Comment:
All of this change is just removing this ``no_op()`` context, that as it says, does nothing.